packages/core/blueprints/navi-core/index.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  normalizeEntityName() {}, // no-op since we're just adding dependencies
+
+  afterInstall() {
+    return this.addAddonsToProject({
```

Comment:
Do addons typically print any messages here? Not necessary but could be nice

Reply:
`addAddonsToProject` will print the addons it adds. Did you have something else in mind?

Reply:
Nope just wanted to check it was printed somewhere